diff/parser/parser.go

```diff
@@ -90,19 +98,90 @@ NextToken:
 					case "lon":
 						node.Long, _ = strconv.ParseFloat(attr.Value, 64)
 					}
+
+					if config.ParseMetadata {
+						switch attr.Name.Local {
+						case "id":
```

Comment:
I hope this is solved with https://github.com/ImreSamu/imposm3/commit/771e7f527840d49897b6457defedc455c6797c74